test/support/stub_model_filters.rb

```diff
@@ -0,0 +1,5 @@
+module StubModelFilters
+  def stub_filter(name)
+    define_singleton_method(name) { |*| nil }
+  end
+end
```

Comment:
I think this could live within the single file that uses it, no need for a new support file. Wdyt?